Week1/homework/js-exercises/logNumber.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+
+var numberX;
+console.log("The value of X is undefined.");
+console.log(numberX);
+numberX=26;
```

Comment:
`let` here please instead of `var`. Also, a minor spacing problem here in line 7.